File: java/maven_bdd/specs2/src/main/java/fits/sample/Book.java
package fits.sample;

import java.util.ArrayList;
import java.util.List;

public class Book {
	private String title;
	private List<Comment> comments;

	public Book() {
		this.comments = new ArrayList<Comment>();
	}

	public String getTitle() {
		return this.title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public List<Comment> getComments() {
		return this.comments;
	}
}

File: scala/morphia/sample1/src/main/scala/Book.scala
package fits.sample

import java.util.{List, ArrayList}
import com.google.code.morphia.annotations._
import org.bson.types.ObjectId

@Entity
class Book(@Property var title: String, @Property var isbn: String) {

	//デフォルトコンストラクタは必須
	def this() = this("", "")

	@Id
	var id: ObjectId = null

	@Embedded
	var comments: List[Comment] = new ArrayList[Comment]()

}